CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v0.42.2
+------------------------------
+*April 30, 2018*
+
+### Fixed
+- Broken build is fixed
```

Comment:
Nit – doesn't actually say what's been fixed 😉 

Can you add that it was depending on an icon that isn't always present in the ratings